tools/pipelines/test-coverage.yml

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# test-coverage pipeline (for the purpose of detecting flaky tests)
+
+name: $(Build.BuildId)
+
+trigger: none
+pr: none
+
+schedules:
+  - cron: "00 03 * * SAT"
```

Comment:
Since UTC is used, this will actually run Friday evening Pacific. Postpone this about five hours later, just to be safe. Or Sunday.